editoast/map_layers.yml

```
@@ -128,6 +128,16 @@ layers:
         joins:
           - inner join infra_object_neutral_section neutral_section on neutral_section.obj_id = layer.obj_id and neutral_section.infra_id = layer.infra_id
 
+  level_crossings:
+    table_name: infra_layer_level_crossing
+    id_field: id
+    views:
+      geo:
+        on_field: geographic
+        data_expr: level_crossing.data
```

Maybe we can lighter the payload removing `parts`.
I'm not 100% sure the generated sql query is valid but we can try it out. If it does not work, we can keep it this way.

```suggestion
        data_expr: level_crossing.data - 'parts'
```

---

Done